src/test/java/org/folio/processing/mapping/mapper/writer/marc/MarcBibRecordModifierTest.java

```diff
@@ -67,7 +68,7 @@ public void shouldRemoveLinksOnRepeatableLinkedFieldRemoval() throws IOException
   public void shouldNotUpdateLinkedSubfields() throws IOException {
     // given
     var incomingParsedContent = "{\"leader\":\"00049nam  22000371a 4500\",\"fields\":[{\"001\":\"ybp7406411\"}," +
-      "{\"100\":{\"subfields\":[{\"a\":\"electronic updated\"},{\"0\":\"test\"},{\"9\":\"aabf59b7-913b-42ac-b1c6-e50ae7b00e6a\"}],\"ind1\": \" \",\"ind2\":\" \"}}," +
+      "{\"100\":{\"subfields\":[{\"a\":\"electronic updated\"},{\"0\":\"test\"},{\"9\":\"bdbf59b7-913b-42ac-b1c6-e50ae7b00e6a\"}],\"ind1\": \" \",\"ind2\":\" \"}}," +
```

Comment:
This $9 was set to "aabf59b7-913b-42ac-b1c6-e50ae7b00e6a" intentionally, to check existing is not being replaced.
I'd better leave it as "aabf59b7-913b-42ac-b1c6-e50ae7b00e6a"